### PR TITLE
SimpleTokenParser service

### PR DESCRIPTION
**Pull request by m-vo**
This documents how to use/extend the simple token parser service (compare https://github.com/contao/contao/pull/1756).

It's no full reference by any means. 
But at least, this way I won't have forgotten how extending works in a few weeks. :upside_down_face: 

/cc @Toflar

### CODE REVIEW

**Review by Toflar — COMMENTED** [review]
Very good! The version 4.10 hint is missing though :)

**Review by fritzmg — APPROVED** [review]

File: docs/dev/reference/services.md [hunk 1]
```diff
@@ -136,6 +136,49 @@ class Example
 ```
 
 
+## SimpleTokenParser
+
```
Comment (fritzmg) [inline docs/dev/reference/services.md:140]: As mentioned by @Toflar , pls add `{{< version "4.10" >}}` here :)

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/dev/reference/services.md (hunk 1 above)
Comment (m-vo) [reply docs/dev/reference/services.md:140]: Done in 0bce42b6fb9e29add028c3d63e072499dec33db6

**Comment by m-vo** [comment]
You guys are fast :smile:

**Review by fritzmg — APPROVED** [review]

**Review by Toflar — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @m-vo :)

**Pull request merged**